indico/web/forms/widgets.py

```diff
@@ -159,3 +159,33 @@ def __call__(self, field, **kwargs):
 
         options.update(kwargs.pop('options', {}))
         return super(SelectizeWidget, self).__call__(field, options=options)
+
+
+class StaticSelectizeWidget(JinjaWidget):
+
+    def __init__(self, choices=None):
+        if not choices or not isinstance(choices, list):
```

Comment:
`isinstance` checks are not very pythonic and i don't really see the value of it here.

I'd go for a simple

    self.choices = list(choices) if choices is not None else []